src/metpy/plots/declarative.py

```diff
@@ -1173,11 +1173,18 @@ def griddata(self):
                 if selector is not None:
                     subset[dim_coord] = selector
             data_subset = data.metpy.sel(**subset).squeeze()
-            if data_subset.ndim != 2:
-                raise ValueError(
-                    'Must provide a combination of subsetting values to give 2D data subset '
-                    'for plotting'
-                )
+            if (data_subset.ndim != 2):
+                if data_subset.ndim == 3:
+                    if (data_subset.shape[-1] not in [3, 4]):
```

Comment:
Would prefer a tuple to list here:
```suggestion
                    if (data_subset.shape[-1] not in (3, 4)):
```

I wouldn't hold this up for this but...

Reply:
Easy enough, is there a different overhead for creating the list vs. tuple or the immutable nature of tuples, or just a pure preference? Just curious as I tend to default to a list for most of these types of things.

Reply:
Philosophically, lists imply some kind of meaningful order; when you just have a group (pair, triplet, quartet, etc.) of items that you aren't ordering (or otherwise modifying the collection, best to default to tuple.

The immutable nature of tuples makes them more performant:
<img width="693" alt="image" src="https://user-images.githubusercontent.com/221526/166783940-b70c72e9-b492-4bff-a63a-3f4625758c42.png">

so 2x as fast, though admittedly we're talking saving 25ns.